RIOT-OS/RIOT
# dist/tools/vera++: improve C/C++ support and optimize CI call

### Contribution description
This commit solves some issues and optimizes the check script in order to run Vera++ only once (instead of twice, one for warnings and one for errors).

In concrete, this PR:
- Adds a "max-line-length-warn" in L004 (Line too long) to warn when line length > 80. This paremeter is configurable from `riot_params.txt`. Note that this warning is only a "puts" command, so it won't make Vera++ return error code
- Modifies T003 (some keywoards should be followed by space) in order to differentiate between C and C++ keywords. This should solve  https://github.com/RIOT-OS/RIOT/issues/15626
- Removes the first call to Vera++ (for warnings) and remove unused files.



### Testing procedure
- Check that Vera++ doesn't show false positives (see https://github.com/RIOT-OS/RIOT/issues/15626)
- Check that the warnings for lines > 100 chars are still shown.



### Issues/PRs references
Fixes https://github.com/RIOT-OS/RIOT/issues/15626

_I hope I don't have to touch TCL files again in my whole life_
